### DISCUSSION TITLE
Domain name

### DISCUSSION THREAD

**Discussion by cholakovit** [vercel/community#3579] [Help]
### Summary

Hi, I redirected one domain name to my hosting account at vercel that at the moment is free and i redircted a second domain. The first one worked, but the second one is not visible in my account at vercel. I double checked the DNS and they are right. They are pointing at vercel, but the second domain doesn't appear in my account. Help ?

### Steps to Reproduce

Hi, I redirected one domain name to my hosting account at vercel that at the moment is free and i redircted a second domain. The first one worked, but the second one is not visible in my account at vercel. I double checked the DNS and they are right. They are pointing at vercel, but the second domain doesn't appear in my account. Help ?

**Comment by amyegan** [comment]
Hey @cholakovit. Did you already [add the custom domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain) in project settings?
